src/app/(proper_react)/(redesign)/(public)/LandingView.test.tsx

```diff
@@ -81,6 +82,34 @@ it("passes the user's email address to the identity provider", async () => {
   );
 });
 
+it("sends telemetry when a free scan CTA is shown in the viewport", async () => {
+  const mockedRecord = useTelemetry();
+  const ComposedDashboard = composeStory(LandingNonUs, Meta);
+  render(<ComposedDashboard />);
+
+  // jsdom will complain about not being able to navigate to a different page
+  // after clicking the link; suppress that error, as it's not relevant to the test:
+  jest.spyOn(console, "error").mockImplementation(() => undefined);
+
+  // The useViewTelemetry ref is attached to the form, not the button
+  const submitButton = screen.getAllByRole("button", {
+    name: "Get free scan",
+  });
+  const form = submitButton[0].closest("form");
+  expect(form).toBeInTheDocument();
+
+  // Trigger intersection on the form element
+  await act(async () => {
+    mockIsIntersecting(form!, true);
+  });
+
+  expect(mockedRecord).toHaveBeenCalledWith(
+    "ctaButton",
+    "view",
+    expect.objectContaining({ button_id: "clicked_get_scan_header" }),
+  );
+});
```

Comment:
I traced this component back to [https://github.com/mozilla/blurts-server/pull/4794/files](https://github.com/mozilla/blurts-server/pull/4794/files), so I rewrote the test to only account for the “Get Free Scan” non-US state.